### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by molly-moen**
# Type of change? 

- [ ] Bug fix
- [ ] New feature
- [x] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
Update README to be in more informative and more in line with template.

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,15 +43,41 @@ Build, run and test locally
 * `npm test`: Runs test in watch mode, press `q` to quit
 * `npm flow`: Runs Flow type checker. Learn more about Flow [here](https://flow.org/).
```
[the remark this answers is not in the document]
Comment (molly-moen) [reply README.md]: npm start is after npm install in the list

File: README.md [hunk 2]
```diff
@@ -14,15 +14,27 @@ Web app for Azure IoT PCS Remote Monitoring Solution [dotnet](https://github.com
...
 1. Install [node.js](https://nodejs.org/)
 2. For development, you can use your preferred editor
    - [Visual Studio Code](https://code.visualstudio.com/)
    - [Atom](https://atom.io/)
    - [Sublime Text](https://www.sublimetext.com/)
    -  or other preferred editor
 
-Build, run and test locally
+### 3. Environment variables required to run the service
+In order to run the service, the environment variables need to be created at least once. More information on configuring environment variables [here](#configuration-and-environment-variables).
```
[the remark this answers is not in the document]
Comment (molly-moen) [reply README.md]: We do deploy it via docker as a microservice? Jill/Parvez/Stephen is there a better term?

File: README.md [hunk 3]
```diff
@@ -51,9 +89,15 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 You can find a guide to using it [here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md).
 
+Other Technologies Used
```
Comment (spryor) [inline README.md]: I would replace this section with this:

```
Core technologies overview
==============================
- [ReactJS](https://reactjs.org/)
- [React-Router v4](https://github.com/ReactTraining/react-router)
- [Redux](https://redux.js.org/)
- [Redux-Observable](https://redux-observable.js.org/)
- [RxJs](http://reactivex.io/rxjs/)
- [SASS](http://sass-lang.com/)
- [React-i18nnext](https://github.com/i18next/react-i18next)
```

File: README.md [hunk 4]
```diff
@@ -31,15 +43,41 @@ Build, run and test locally
...
-Environment variables are configured in the [.env](.env) file. If you would like to configure the service to use local services you can include values for the following environment variables
-- REACT_APP_BASE_SERVICE_URL
-- REACT_APP_IOTHUBMANAGER_WEBSERVICE_PORT
-- REACT_APP_DEVICESIMULATION_WEBSERVICE_PORT
-- REACT_APP_CONFIG_WEBSERVICE_PORT
-- REACT_APP_TELEMETRY_WEBSERVICE_PORT
-- REACT_APP_AUTH_WEBSERVICE_PORT
+Project Structure
+===========================
+This microservice contains the following sections under [src](src):
```
[the remark this answers is not in the document]
Comment (ppathan) [reply README.md]: @molly-moen I think instead of microservice it should be webui or web app

File: README.md [hunk 5]
```diff
@@ -14,14 +14,26 @@ Web app for Azure IoT PCS Remote Monitoring Solution [dotnet](https://github.com
```
Comment (ppathan) [inline README.md]: Probably right time to start dropping PCS from the repos and readmes. We can start from here. `Azure IoT Remote Monitoring Solution`

File: README.md [hunk 6]
```diff
@@ -14,14 +14,26 @@ Web app for Azure IoT PCS Remote Monitoring Solution [dotnet](https://github.com
 
 Prerequisites
 =============
-## Setup Node
+### 1. Deploy a Remote Monitoring Solution
+The easiest way to test the Web UI is against a deployed remote monitoring solution. The solution can be deployed via the [web interface](https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-remote-monitoring-deploy) or via the [command line](https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-remote-monitoring-deploy-cli).
+
```
Comment (ppathan) [inline README.md]: You can remove the `en-us` from links.

File: README.md [hunk 7]
```diff
@@ -14,14 +14,26 @@ Web app for Azure IoT PCS Remote Monitoring Solution [dotnet](https://github.com
 
 Prerequisites
 =============
-## Setup Node
+### 1. Deploy a Remote Monitoring Solution
+The easiest way to test the Web UI is against a deployed remote monitoring solution. The solution can be deployed via the [web interface](https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-remote-monitoring-deploy) or via the [command line](https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-remote-monitoring-deploy-cli).
+
+It is also possible to [deploy the solution locally](https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-remote-monitoring-deploy-local#deploy-the-azure-services).
+
```
Comment (ppathan) [inline README.md]: `en-us`

**Pull request merged**